### PR TITLE
WIP BXMSDOC-3835: Testing a decision service using test scenarios for PAM/DM 7.3

### PR DESCRIPTION
**Pull request by gadeyredhat**
[Jira Id](https://issues.jboss.org/browse/BXMSDOC-3835)

[PAM Rendered Output](http://file.pnq.redhat.com/~gadey/BXMSDOC-3835-pam/)
[DM Rendered Output](http://file.pnq.redhat.com/~gadey/BXMSDOC-3835-dm/)

### CODE REVIEW

File: assemblies/assembly_test-scenarios/main.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ include::_artifacts/document-attributes.adoc[]
 include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
-As a business analyst or business rules developer, you can use test scenarios in {CENTRAL} to test a decision service before a project is deployed. Testing a decision service ensures that rule assets in the project function properly and as expected. You can test a decision service at any time during project development.
+As a business analyst or business rules developer, you can use test scenarios in {CENTRAL} to test a decision service before a project is deployed. You can test both DMN-based decision service or rules-based decision service and ensure that all assets in the project are functioning properly and as expected. Also, you can test a decision service at any time during project development.
```
Comment (csherrar) [inline assemblies/assembly_test-scenarios/main.adoc]: Minor suggestion:

s/You can test both DMN-based decision service or rules-based decision service and ensure that all assets in the project are functioning properly and as expected./You can test DMN-based and rules-based decision services to ensure these are functioning properly and as expected.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenario-designer-con.adoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[id='test-scenario-designer-con']
+= Using the Test Scenario (Legacy) designer in {CENTRAL}
+
+In {PRODUCT}, there are two test scenarios designers. One is the *Test Scenario* designer and the other is *Test Scenario (Legacy)* designer. You can use either of them to test your business rules & data. The *Test Scenario* designer is the default designer and you can create and run DMN-based test scenarios using it. This chapters covers how to use the *Test Scenario (Legacy)* designer to create and run test scenarios. The *Test Scenario (Legacy)* designer runs test scenario based on DRL files and Guided Decision Tables.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenario-designer-con.adoc]: Minor suggestion - using the more common word "describes" and pluralizing "scenarios":

s/covers/describes

s/The *Test Scenario (Legacy)* designer runs test scenario based on DRL files and Guided Decision Tables./The *Test Scenario (Legacy)* designer runs test scenarios based on DRL files and Guided Decision Tables.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc [hunk 3]
```diff
@@ -4,10 +4,10 @@
 The *EXPECT* section defines the expected results based on the *GIVEN* input facts. That is, *GIVEN* the input facts, *EXPECT* other specified facts to be valid or entire rules to be activated. For example, with the given facts of an applicant under the age of 21 in the scenario, the *EXPECT* results could be `LoanApplication` with `approved` set to `false` (as a result of the underage applicant), or could be the activation of the `Underage` rule as a whole.
 
 .Prerequisite
-All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the test scenarios designer.
+All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the legacy test scenarios designer.
 
 .Procedure
-. In the test scenarios designer, click *EXPECT* to open the *New expectation* window with the available facts.
+. In the legacy test scenarios designer, click *EXPECT* to open the *New expectation* window with the available facts.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc]: Minor suggestion - to make the reference consistent. Maybe check if there is a label in the UI?

- legacy test scenarios designer
- *Test Scenario (Legacy)* designer

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc [hunk 4]
```diff
@@ -4,10 +4,10 @@
 The *EXPECT* section defines the expected results based on the *GIVEN* input facts. That is, *GIVEN* the input facts, *EXPECT* other specified facts to be valid or entire rules to be activated. For example, with the given facts of an applicant under the age of 21 in the scenario, the *EXPECT* results could be `LoanApplication` with `approved` set to `false` (as a result of the underage applicant), or could be the activation of the `Underage` rule as a whole.
 
 .Prerequisite
-All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the test scenarios designer.
+All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the legacy test scenarios designer.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc]: Minor suggestion - to make the reference consistent. Maybe check if there is a label in the UI?

- legacy test scenarios designer
- *Test Scenario (Legacy)* designer

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc [hunk 5]
```diff
@@ -4,10 +4,10 @@
 The *GIVEN* section defines input facts for the test. For example, if an `Underage` rule in the project declines loan applications for applicants under the age of 21, then the *GIVEN* facts in the test scenario could be `Applicant` with `age` set to some integer less than 21.
 
 .Prerequisite
-All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the test scenarios designer.
+All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the legacy test scenarios designer.
 
 .Procedure
-. In the test scenarios designer, click *GIVEN* to open the *New input* window with the available facts.
+. In the legacy test scenarios designer, click *GIVEN* to open the *New input* window with the available facts.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc]: Minor suggestion - to make the reference consistent. Maybe check if there is a label in the UI?

- legacy test scenarios designer
- *Test Scenario (Legacy)* designer

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc [hunk 6]
```diff
@@ -4,10 +4,10 @@
 The *GIVEN* section defines input facts for the test. For example, if an `Underage` rule in the project declines loan applications for applicants under the age of 21, then the *GIVEN* facts in the test scenario could be `Applicant` with `age` set to some integer less than 21.
 
 .Prerequisite
-All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the test scenarios designer.
+All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the legacy test scenarios designer.
```
Comment (csherrar) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc]: Minor suggestion - to make the reference consistent. Maybe check if there is a label in the UI?

- legacy test scenarios designer
- *Test Scenario (Legacy)* designer

File: doc-content/enterprise-only/project-data/test-designer-alias-con.adoc [hunk 7]
```diff
@@ -0,0 +1,4 @@
+[id='test-designer-alias-con']
+= Concept of aliases (for rule-based scenarios only)
+
+In the Test Scenario designer, once you map a header cell with a data object, the data object is removed from the *Test Editor* tab in the *Test Tools* panel. You can re-map a data object to another header cell by using the concept of *Alias*. Aliases enable you to specify multiple instances of the same data object. Ensure that the aliases are uniquely named. By doing so, the data object again becomes available in the *Test Editor* tab for remapping. To introduce an alias, double click on a corresponding header cell and manually change the name. You can create property aliases as well. This allows you to rename the used properties directly in the table which enhances user experience.
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-alias-con.adoc]: Minor suggestion - remove "which enhances user experience.":

s/This allows you to rename the used properties directly in the table which enhances user experience./This allows you to rename the used properties directly in the table.

File: doc-content/enterprise-only/project-data/test-designer-con.adoc [hunk 8]
```diff
@@ -0,0 +1,4 @@
+[id='test-designer-con']
+= Test Scenario designer
+
+{PRODUCT} now comes with a new *Test Scenario* designer which fully supports DMN model testing. This designer provides a new and enhanced user experience when you test or validate DMN-based test scenarios. The aim of the new designer is to help the user test the proper functioning of the created rules and knowledge bases.
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-con.adoc]: Minor suggestion:

s/which/that

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc [hunk 9]
```diff
@@ -0,0 +1,34 @@
+[id='test-designer-create-test-template-dmn-based-proc']
+= Creating a test scenario template for DMN-based test scenarios
+
+Create a test scenario template for DMN-based scenarios by following the procedure below to validate your rules and data.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project for which you want to create the test scenario.
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Minor Suggestion - "for which" or "that ... for":

s/In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project for which you want to create the test scenario./In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project that you want to create the test scenario for.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+. Click *Add Asset* -> *Test Scenario*.
+. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets have been assigned or will be assigned.
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Minor suggestion - shorten sentence and simplify:

. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets have been assigned or will be assigned./. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets assigned.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+. Select *DMN* as the *Source type*.
+. Select an existing DMN asset using the *Choose DMN asset* option.
+. Click *Ok* to create and open the test scenario in the *Test Scenario* designer.
+. To map the *GIVEN* column header to a data object,
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Minor suggestion - replace comma with colon:

s/. To map the *GIVEN* column header to a data object,/. To map the *GIVEN* column header to a data object:

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+.. Click an instance header in the *GIVEN* section.
+.. Select the data object from the *Test Editor* tab.
+.. Click *Add*.
+. To insert more properties of the data object, right-click the property header and select *Insert column right* or *Insert column left* as required.
+. To map a data object field to a property cell,
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Minor suggestion - replace comma with colon

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+.. Click a property cell.
+.. Select the data object field from the *Test Editor* tab.
+.. Click *Add*.
+. To map the *EXPECTED* column header to a data object,
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Minor suggestion - replace comma with colon

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+.. Click an instance header in the *EXPECTED* section.
+.. Select the data object from the *Test Editor* tab.
+.. Click *Add*.
+. To insert more properties of the data object, right-click the property header and select *Insert column right* or *Insert column left* as required.
+. To map a data object field to a property cell,
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Minor suggestion - replace comma with colon

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+.. Click a property cell.
+.. Select the data object field from the *Test Editor* tab.
+.. Click *Add*.
+
+Use the contextual menu to add or remove columns and rows as needed.
+
+After you have created and mapped both the *GIVEN* & *EXPECTED* columns to a data object & its properties, you have to define the test scenario which comes next.
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Minor suggestion - simplify and replace "&" with "and":

s/After you have created and mapped both the *GIVEN* & *EXPECTED* columns to a data object & its properties, you have to define the test scenario which comes next./After you create and map both the *GIVEN* and *EXPECTED* columns to a data object and its properties, you then have to define the test scenario.

File: doc-content/enterprise-only/project-data/test-designer-delete-test-proc.adoc [hunk 16]
```diff
@@ -1,10 +1,10 @@
-[id='preview-editor-delete-test-proc']
+[id='test-designer-delete-test-proc']
 = Deleting a test scenario
 
-You can delete existing test scenarios which were created using the *Test Scenario (Preview)* editor.
+You can delete existing test scenarios which were created using the *Test Scenario* designer.
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/test-designer-delete-test-proc.adoc]: Minor suggestion - replace "which" with "that":

s/You can delete existing test scenarios which were created using the *Test Scenario* designer./You can delete existing test scenarios that were created using the *Test Scenario* designer.

**Review by csherrar — COMMENTED** [review]
Just some minor suggestions - otherwise looks good to me.

File: assemblies/assembly_test-scenarios/main.adoc (hunk 1 above)
Comment (gadeyredhat) [reply assemblies/assembly_test-scenarios/main.adoc]: Changed as suggested.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenario-designer-con.adoc (hunk 2 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenario-designer-con.adoc]: Changed as suggested.

File: assemblies/assembly_test-scenarios/main.adoc (hunk 1 above)
Comment (gadeyredhat) [reply assemblies/assembly_test-scenarios/main.adoc]: Changed as suggested

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc (hunk 4 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc]: Changed to UI label as suggested.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc (hunk 3 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc]: Changed to UI label as suggested.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc (hunk 6 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc]: Changed to UI label as suggested.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc (hunk 5 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc]: Changed to UI label as suggested.

File: doc-content/enterprise-only/project-data/test-designer-alias-con.adoc (hunk 7 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-alias-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-con.adoc (hunk 8 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-con.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc (hunk 9 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc (hunk 10 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc (hunk 11 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc (hunk 12 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc (hunk 13 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc (hunk 14 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc (hunk 15 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/project-data/test-designer-delete-test-proc.adoc (hunk 16 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/project-data/test-designer-delete-test-proc.adoc]: Changed as suggested.

**Comment by gadeyredhat** [comment]
Made all changes as suggested by @csherrar for peer review.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-basic-operations-con.adoc [hunk 17]
```diff
@@ -1,7 +1,7 @@
-[id='preview-editor-basic-operations-con']
-= Basic operations using the Test Scenario (Preview) editor
+[id='test-designer-basic-operations-con']
+= Basic operations using the Test Scenario designer
```
Comment (danielezonca) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-basic-operations-con.adoc:2]: Why `designer` now? I prefer `editor` personally (this comment is valid for the entire doc of course).

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-scenario-template-con.adoc [hunk 18]
```diff
@@ -0,0 +1,6 @@
+[id='test-designer-create-test-scenario-template-con']
+= Test scenario template
+
+Before specifying test scenario definitions, you need to create a test scenario template. The header of the test scenario table defines the template for each scenario. You need to set the types of the instance and property headers for both the GIVEN and EXPECTED sections. Instance headers map to a particular data object (a fact), whereas the property headers map to a particular field of the corresponding data object.
```
Comment (danielezonca) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-scenario-template-con.adoc]: It is `EXPECT` not `EXPECTED`. Please update the entire doc

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-template-dmn-based-proc.adoc [hunk 19]
```diff
@@ -0,0 +1,34 @@
...
+Create a test scenario template for DMN-based scenarios by following the procedure below to validate your rules and data.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project that you want to create the test scenario for.
+. Click *Add Asset* -> *Test Scenario*.
+. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets assigned.
+. Select *DMN* as the *Source type*.
+. Select an existing DMN asset using the *Choose DMN asset* option.
+. Click *Ok* to create and open the test scenario in the *Test Scenario* designer.
+. To map the *GIVEN* column header to a data object:
```
Comment (danielezonca) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-template-dmn-based-proc.adoc]: Starting from this point the rest of the chapter is not correct. The scenario template of a DMN scenario is automatically generated during file creation based on DMN model: each input node becomes one or more GIVEN columns and each decision node becomes one or more EXPECT columns.
What you are describing here is correct but it is useful only for edit the template: user can decide to remove some columns and then add them again (for instance to move a decision as input)

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-expressions-syntax-dmn-based-ref.adoc [hunk 20]
```diff
@@ -0,0 +1,33 @@
+[id='test-designer-expressions-syntax-dmn-based-ref']
+= Expression syntax in DMN-based scenarios
+The following data types are supported by the DMN-based test scenarios in the *Test Scenario* designer:
```
Comment (danielezonca) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-expressions-syntax-dmn-based-ref.adoc:3]: I think it should be useful to mention also here that we support FEEL as language. The rest of the paragraph is just an example of FEEL data types / expressions

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-create-proc.adoc [hunk 21]
```diff
@@ -11,9 +11,9 @@ With this data, the test scenario can validate the expected and actual results f
 
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project name.
-. Click *Add Asset* -> *Test Scenario*.
-. Enter an informative *Test Scenario* name and select the appropriate *Package*. The package that you specify must be the same package where the required rule assets have been assigned or will be assigned. You can import data objects from any package into the asset's editor.
-+
+. Click *Add Asset* -> *Test Scenario (Legacy)*.
+. Enter an informative *Test Scenario* name and select the appropriate *Package*. The package that you specify must be the same package where the required rule assets have been assigned or will be assigned. You can import data objects from any package into the asset's designer.
+. Select *RULE* as the *Source type*.
```
Comment (danielezonca) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-create-proc.adoc]: This is not correct. There is no `Source type` option in the legacy test scenario

**Review by danielezonca — CHANGES_REQUESTED** [review]

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-create-proc.adoc (hunk 21 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-create-proc.adoc]: Corrected as suggested.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-expressions-syntax-dmn-based-ref.adoc (hunk 20 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-expressions-syntax-dmn-based-ref.adoc:3]: @danielezonca, in the rendered output, the para just before this section mentions that - "While rule-based test scenarios support a basic expression language, DMN-based test scenarios support the FEEL expression language.". So I believe we should not mention the same thing again here. Hence, no change.

Comment (danielezonca) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-expressions-syntax-dmn-based-ref.adoc:3]: Ok in this case, fine for me

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-scenario-template-con.adoc (hunk 18 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-scenario-template-con.adoc]: Corrected the naming of the UI element to *EXPECT* as suggested.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-basic-operations-con.adoc (hunk 17 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-basic-operations-con.adoc:2]: As per a recent doc team's decisions, it was decided ti use the word *Designer* in place of "Editor" going forward. I believe Stetson has communicated this to Klara as well. Hence no change to this suggestion.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-template-dmn-based-proc.adoc (hunk 19 above)
Comment (gadeyredhat) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-template-dmn-based-proc.adoc]: @danielezonca, I have made the changes as per Klara's feedback in the QE review. Refer to [Klara's QE review](https://issues.jboss.org/browse/BXMSDOC-3836) for more details.

**Pull request merged**